### PR TITLE
fix: Fixing and sharpening text in glossary.

### PR DESCRIPTION
**Pull request by stoobie**

### CODE REVIEW

File: modules/ROOT/partials/partial_glossary.adoc [hunk 1]
```diff
@@ -81,7 +79,7 @@ A single transaction that executes multiple bundled transactions, which are sign
 [#prover]
 == prover
 
-An entity that receives the output of Cairo programs and generates proofs to be verified. The prover submits the proof to the verifier that registers the fact on L1. Their main task is to validate the work of athe sequencers when they receive the block produced by the sequencer, and to generate proofs that these processes were correctly performed.
+An entity that receives the output of Cairo programs and generates proofs to be verified. The prover submits the proof to the verifier that registers the fact on L1. The prover generates proofs of computational integrity of the work performed by the sequencer.
```
Comment (stoobie) [inline modules/ROOT/partials/partial_glossary.adoc]:
```suggestion
The entity that receives the output of Cairo programs and generates proofs to be verified. In the context of Starknet, the prover generates proofs of the computational integrity of the work performed by the sequencer. It then submits the proof to the verifier on L1, which registers the fact.
```

File: modules/ROOT/partials/partial_glossary.adoc [hunk 2]
```diff
@@ -25,28 +25,26 @@ The Starknet Node API uses the Remote Procedure Call (RPC) architectural style,
...
+block hash:: A unique identifier that can be used to query and refer to the block.
 block number:: A number representing the location of the block in the sequence of accepted blocks in Starknet.
 
 [#builtin]
 == builtin
 
-{Cairo_header}
-
-A predefined optimized low-level execution unit that is added to the Cairo CPU board to perform predefined computations that are expensive to perform in vanilla Cairo, including range-checks, Pedersen hash calculations, and ECDSA.
+A optimized low-level execution unit that is added to the Cairo CPU board to perform predefined computations that are expensive to perform in standard Cairo, including range-checks, Pedersen hash calculations, and ECDSA.
```
Comment (leo-starkware) [inline modules/ROOT/partials/partial_glossary.adoc]: To change 'A' to 'An'

**Review by leo-starkware — COMMENTED** [review]
Left 1 minor comment

File: modules/ROOT/partials/partial_glossary.adoc (hunk 2 above)
Comment (stoobie) [inline modules/ROOT/partials/partial_glossary.adoc]:
```suggestion
An optimized low-level execution unit that is added to the Cairo CPU board to perform predefined computations that are expensive to perform in standard Cairo, including range-checks, Pedersen hash calculations, and ECDSA.
```

Comment (stoobie) [reply modules/ROOT/partials/partial_glossary.adoc]: Fixed.

**Review by leo-starkware — APPROVED** [review]
LGTM

**Pull request merged**